phpBB/language/en/install.php

```diff
@@ -530,6 +532,27 @@
 	'UPDATE_DATABASE_SCHEMA'		=> 'Updating database schema',
 	'UPDATE_FILES'					=> 'Update files',
 	'UPDATE_FILES_NOTICE'			=> 'Please make sure you have updated your board files too, this file is only updating your database.',
+	'UPDATE_INCOMPLETE'				=> 'Your phpBB installation has not been correctly updated.',
```

Comment:
All of this here seems like a big confusing mess of information (to me, and so probably to anybody else). It's also not necessary to have installation instructions in the middle of the installation process.

All you are checking is if the db version is < the package version. If that's true, all that means is the user needs to run the database update script. So just provide a link to the update script at root/install/database_update.php.